test/AutoinstClass_test.rb

```diff
@@ -198,4 +199,174 @@
       expect(subject.classDir).to eq(test_xml_dir)
     end
   end
+
+  describe '#MergeClasses' do
+    let(:base_profile_path) { File.join('test', 'fixtures', 'profiles', 'partitions.xml') }
+    let(:tmp_dir) { File.join(root_path, 'tmp') }
+    let(:expected_xml) { File.read(expected_xml_path) }
+    let(:output_path) { File.join(tmp_dir, 'output.xml') }
+    let(:output_xml) { File.read(output_path) }
+    let(:dontmerge) { [] }
+    let(:merge_xslt_path) { File.join('xslt', 'merge.xslt') }
+    let(:xsltproc_command) {
+      "/usr/bin/xsltproc --novalid --param replace \"'false'\"  " \
+      "--param with \"'#{subject.findPath("largeswap.xml", "swap")}'\"  "\
+      "--output #{File.join(tmp_dir, "output.xml")}  " \
+      "#{merge_xslt_path} test/fixtures/profiles/partitions.xml "
+    }
+
+    around(:each) do |example|
+      FileUtils.mkdir(tmp_dir) unless Dir.exist?(tmp_dir)
```

Comment:
I didn't thought about that but you're right. Anyway, I'll keep doing the rm_rf after the test is run, isn't it? (so in case that everything goes well the tmp dir is cleared).